testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -1763,9 +1763,9 @@ public void testSparkReadsTrinoTableAfterOptimizeAndCleaningUp(StorageFormat sto
 
         onTrino().executeQuery(format("ALTER TABLE %s EXECUTE OPTIMIZE", trinoTableName));
         onTrino().executeQuery("SET SESSION iceberg.expire_snapshots_min_retention = '0s'");
-        onTrino().executeQuery("SET SESSION iceberg.delete_orphan_files_min_retention = '0s'");
+        onTrino().executeQuery("SET SESSION iceberg.remove_orphan_files_min_retention = '0s'");
         onTrino().executeQuery(format("ALTER TABLE %s EXECUTE EXPIRE_SNAPSHOTS (retention_threshold => '0s')", trinoTableName));
-        onTrino().executeQuery(format("ALTER TABLE %s EXECUTE DELETE_ORPHAN_FILES (retention_threshold => '0s')", trinoTableName));
+        onTrino().executeQuery(format("ALTER TABLE %s EXECUTE REMOVE_ORPHAN_FILES (retention_threshold => '0s')", trinoTableName));
```

Comment:
nit: Let's use the lowercased convention for procedure names (as documented in `iceberg.rst`) `REMOVE_ORPHAN_FILES` -> `remove_orphan_files`

Reply:
We do use VACUUM, OPTIMIZE, EXPIRE_SNAPSHOTS so I think REMOVE_ORPHAN_FIILES is perfectly fine here

Reply:
makes sense, but orthogonal